examples/gpu-printing/main.cpp

```diff
@@ -113,9 +113,13 @@ Result execute()
     printBufferViewDesc.type = IResourceView::Type::UnorderedAccess;
     auto printBufferView = gDevice->createBufferView(printBuffer, printBufferViewDesc);
 
+    ITransientResourceHeap::Desc transientResourceHeapDesc = {};
+    transientResourceHeapDesc.constantBufferSize = 256;
+    auto transientHeap = gDevice->createTransientResourceHeap(transientResourceHeapDesc);
```

Comment:
I'm a bit surprised that there is nothing here to indicate the "depth" of the queue of scopes/frames inside the heap.